README.md

```
@@ -1,15 +1,185 @@
+
+### 실시간 프리뷰 업데이트
+- 사용자의 카드 정보 입력에 따라 카드 프리뷰 화면이 실시간으로 동기화
+- 각 입력 필드의 변화가 즉시 카드 이미지에 반영
+
+## 폴더 구조
 
-```sh
-npm create vite@latest my-app -- --template react-ts
-npm install
-npm run dev
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)